### PR TITLE
fixing closed-loop stepper motor alarm (for U4)

### PR DESCRIPTION
**Pull request by LukasBaecker**
Due to the fact that U4 with Tornado has the only closed-loop stepper motor which is not adressed with CAN as later stepper motors. A fix for the internal alarm is necessary. 

- [x] Creating quick fix for U4
- [x] Fixing the invert of the alarm with use of config file to set the hardware when setting up the robot.
- [x] Test on U4
- [x] Test on other robot (should not have an influence)

Test steps:
- [x] Deploy code without updating the config. It should raise a KeyError
- [x] Update the config
- [x] Drive and reference the robot

### CODE REVIEW

**Comment by pascalzauberzeug** [comment]
As expected F13/RB36 runs without any errors

**Comment by pascalzauberzeug** [comment]
This must be tested on U4 and until then U4 must not be newly configured!

File: field_friend/hardware/field_friend_hardware.py [hunk 1]
```diff
@@ -115,6 +115,10 @@ def __init__(self) -> None:
                                        end_stops_on_expander=config_hardware['y_axis']['end_stops_on_expander'],
                                        )
         elif config_hardware['y_axis']['version'] == 'y_axis_stepper':
+            try:
+                alarm_inverted: bool = config_hardware['y_axis']['alarm_inverted']
+            except KeyError:
+                raise KeyError('\'alarm_inverted\' not found in config_hardware[\'y_axis\']. U4 has an inverted motor alarm input. Check your robot\'s setup.')
```
Comment (LukasBaecker) [inline field_friend/hardware/field_friend_hardware.py:121]: I would change the output a bit more unspecified like:

```
\'alarm_inverted\' not found in config_hardware[\'y_axis\']. Robots with a Closed-Loop-Step-Direction-Motor (U4 and before) has an inverted motor alarm input. Check your robot\'s setup.
```

**Review by LukasBaecker — COMMENTED** [review]
Beside the small changes on the error message, your changes look good @pascalzauberzeug 
Already tested the changes on U4. Worked.

File: field_friend/hardware/field_friend_hardware.py (hunk 1 above)
Comment (LukasBaecker) [reply field_friend/hardware/field_friend_hardware.py]: edited it

**Pull request merged**